x-pack/plugins/security_solution/server/lists_integration/endpoint/handlers/exceptions_pre_delete_item_handler.ts

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { EndpointAppContextService } from '../../../endpoint/endpoint_app_context_services';
+import { ExtensionPoint } from '../../../../../lists/server';
+
+export const getExceptionsPreDeleteItemHandler = (
+  endpointAppContext: EndpointAppContextService
+): (ExtensionPoint & { type: 'exceptionsListPreDeleteItem' })['callback'] => {
```

Comment:
I will edit the type here once the individual extension point types are exported out of the Lists plugin server (PR is pending on that)